include/nccl-headers/neuron/net_device.h

```diff
@@ -0,0 +1,32 @@
+/*************************************************************************
```

Comment:
you don't need any of this file.

Reply:
REmoved file and corresponding structs from API